schemas/openshift/cluster-1.yml

```diff
@@ -196,6 +196,10 @@ properties:
         properties:
           soakDays:
             type: integer
+          locks:
```

Comment:
For a while I understood that this was actually the lock that was going to be somehow updated automatically and not the lock name or the lock kind. Can you rename so that it is clearer: `lockNames`, `lockKinds` or even something fancy as `concurrencyDenialKinds` should represent that this is not the actual lock.

Reply:
Not sure what you mean here. The locking mechanism will be implemented in q-r. This is not an ocm feature

Reply:
I'm just saying I find the name `locks` name a bit confusing, just that. Maybe it's just me

Reply:
`mutexes`

Reply:
Cool. The description looks great too! <3